server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/BasicServlet.java

```diff
@@ -51,8 +52,18 @@
 
   abstract protected String getTitle(HttpServletRequest req);
 
+  public void checkIfActive() throws IOException {
+    // If the HighlyAvailableService is not initialized or it's not the active service, throw an exception
+    // to prevent processing of the servlet.
+    if (null == Monitor.HA_SERVICE_INSTANCE || !Monitor.HA_SERVICE_INSTANCE.isActiveService()) {
+      throw new IOException("This is not the active Monitor", new NotActiveServiceException());
```

Comment:
[`HttpServletResponse`][1] has several useful methods for sending redirects, errors, or otherwise manipulating the status.

[1]: http://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html